overlord/snapstate/snapstate_test.go

```diff
@@ -14547,6 +14548,12 @@ func (s *snapmgrTestSuite) TestResolveChannelPinnedTrack(c *C) {
 		// risk only defaults to pinned kernel track
 		{snap: "kernel", new: "stable", exp: "17/stable", kernelTrack: "17"},
 		{snap: "kernel", new: "edge", exp: "17/edge", kernelTrack: "17"},
+		// risk only with regular snap defaults to pinned track
+		{snap: "some-snap", new: "stable", exp: "3.0/stable", snapChannel: "3.0/edge"},
+		{snap: "some-snap", new: "4.0/stable", exp: "4.0/stable", snapChannel: "3.0/edge"},
```

Comment:
Could use additional test case with the `latest` track:
```
{snap: "some-snap", new: "latest/stable", exp: "latest/stable", snapChannel: "3.0/edge"},
```

Reply:
Added.